src/config.ts

```
@@ -0,0 +1,10 @@
+export const JWT_SECRET = 'a-very-bad-and-hardcoded-secret-that-is-easy-to-guess';
```

ESLint

[error] Insert `⏎·`

(prettier/prettier)